src/channel.c

```diff
@@ -1165,6 +1165,7 @@ static int channel_request_auth_agent(LIBSSH2_CHANNEL *channel,
         if(rc == LIBSSH2_ERROR_EAGAIN) {
             _libssh2_error(session, rc,
                            "Would block sending auth-agent request");
+	    return rc;
```

Comment:
This needs spaces for indent.

Reply:
Good call, I think my editor was going full tabs.